src/pages/CharacterDetails/index.tsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { useQuery } from "@apollo/client";
+
+import { GetCharacter } from "../../queries/queries";
+import { IEpisode } from "../../types/interfaces";
+
+import AddFavorites from "../../components/AddFavorites";
+import CharacterDetailCard from "../../components/CharacterDetailCard";
+import EpisodeCard from "../../components/EpisodeCard";
+import ShowCount from "../../components/ShowCount";
+
+import { ReactComponent as Arrow } from "../../assets/arrow.svg";
+const id = window.location.pathname.split("/")[2];
```

Comment:
extract this inside the function

Reply:
is there any particular reason

Reply:
Its usually better practice and prevents confusion about the scope. Although im not sure about if accessing window creates a scoping problem, because if not, its better to declare it outside and use as a global variable to reduce re-renders. But Im not sure about that, just my opinion